hearth/lib/hearth/http/header_list_parser.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'strscan'
+
+module Hearth
+  module Http
+    # @api private
+    module HeaderListParser
+      # parse a list of possibly quoted and escaped string values
+      # Follows:
+      # # [RFC-7230's specification of header values](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2.6).
+      def self.parse_string_list(value)
+        buffer = StringScanner.new(value)
+        parsed = []
+
+        parsed << read_value(buffer) until buffer.eos?
+
+        parsed
+      end
+
+      def self.read_value(buffer)
```

Comment:
Should all of these be private?

Reply:
Yeah - updated. Rubocop had removed the private for those (since private doesn't actually work on def self.method).  Moved to class << self style.